### ISSUE TITLE
Handle image alt text

### ISSUE THREAD

**Issue by kevinashworth** [kevinashworth/new-aws-kevinashworth-com#14]
The main image for a blog post should have alt text as an entry in the post's front matter.

Currently, the alt text is derived from the post's title or the post's excerpt. This change will allow it to be customized, but would have the same fallbacks.
